### PR TITLE
APEXMALHAR-2518 Terminating operator when there is a server error in processing commit offsets

### PR DESCRIPTION
**Pull request by pramodin**

### CODE REVIEW

**Comment by pramodin** [comment]
@sandeshh please see

**Comment by sandeshh** [comment]
LGTM,  will wait for 2 days for others in the community.

File: kafka/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java [hunk 1]
```diff
@@ -409,6 +418,7 @@ public void onComplete(Map<TopicPartition, OffsetAndMetadata> map, Exception e)
       logger.debug("Commit offsets complete {} ", Joiner.on(';').withKeyValueSeparator("=").join(map));
```
Comment (sandeshh) [inline kafka/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java:419]: Not directly related to your change, Having @Override keyword for onComplete helps.

Comment (pramodin) [reply kafka/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java:419]: Added and rebased

**Comment by sandeshh** [comment]
test this please

**Pull request merged**